Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.mm

```diff
@@ -994,6 +1029,81 @@ - (void)_processMediaInfoDictionaries:(NSArray *)infos atIndex:(NSUInteger)index
     [self _uploadItemFromMediaInfo:info successBlock:uploadItemSuccessBlock failureBlock:failureBlock];
 }
 
+- (void)_processPickerResults:(NSArray<PHPickerResult *> *)results atIndex:(NSUInteger)index manager:(NSFileManager *)manager coordinator:(NSFileCoordinator *)coordinator processedResults:(NSMutableArray<_WKFileUploadItem *> *)processedResults successBlock:(void (^)(NSArray<_WKFileUploadItem *> *processedResults))successBlock failureBlock:(void (^)(void))failureBlock
+{
+    NSUInteger count = [results count];
+    if (index == count) {
+        successBlock(processedResults);
+        return;
+    }
+
+    PHPickerResult *result = [results objectAtIndex:index];
+    ASSERT(index < count);
+    index++;
+
+    auto uploadItemSuccessBlock = ^(_WKFileUploadItem *uploadItem) {
+        [processedResults addObject:uploadItem];
+
+        [self _processPickerResults:results atIndex:index manager:manager coordinator:coordinator processedResults:processedResults successBlock:successBlock failureBlock:failureBlock];
```

Comment:
Hm...it's not super clear what keeps `manager` and `coordinator` alive in this block.